harness/ts/main.ts

```diff
@@ -76,15 +77,17 @@ async function run() {
       });
       await runner.run();
     } catch (err) {
-      console.error(`Feature ${featureDir} failed with ${err}`, (err as any).stack);
+      const errstr = `Feature ${featureDir} failed with ${err}`;
+      failedFeaturesStr += errstr + '\n';
+      console.error(errstr, (err as any).stack);
```

Comment:
This should do it.
```suggestion
      console.error(`Feature ${featureDir} failed`, err);
```

Reply:
The point here was to include each failure in the summary at the end of the CI log where people will see it

Reply:
👍